src/directives/transfer-dom.js

```diff
@@ -1,22 +1,65 @@
 // Thanks to: https://github.com/airyland/vux/blob/v2/src/directives/transfer-dom/index.js
 // Thanks to: https://github.com/calebroseland/vue-dom-portal
 
+import { nanoid } from "nanoid";
+import Vue from "vue";
+
+function createMfePortalElement(guid) {
+    const mfeKey = Vue.prototype.$mfeKey;
+    const portalId = getMfePortalId(guid);
+    if (!mfeKey || !portalId) {
+        return null;
+    }
+
+    const newPortal = document.createElement("div");
+    newPortal.id = portalId;
+    newPortal.classList = [mfeKey];
+    document.body.appendChild(newPortal);
+
+    return newPortal;
+}
+
+function getMfePortalElement(guid) {
+    const portalId = getMfePortalId(guid);
+    return document.getElementById(portalId)
+}
+
+function getMfePortalId(guid) {
+    return `mfe-portal-${guid}`;
+}
+
 /**
  * Get target DOM Node
  * @param {(Node|string|Boolean)} [node=document.body] DOM Node, CSS selector, or Boolean
  * @return {Node} The target that the el will be appended to
  */
-function getTarget (node) {
```

Comment:
The original implementation of this `getTarget` method was pretty weird. I've kept all the checks that it was doing for backwards compatibility (apart of the the instance check)